project/Dependencies.scala

```diff
@@ -1,27 +1,29 @@
-import play.core.PlayVersion.current
-import sbt.*
+import sbt._
+
 
 object Dependencies {
 
-  val playVersion = "play-28"
-  val bootstrapVersion = "7.22.0"
-  val hmrcMongoVersion = "1.3.0"
+  val playVersion: String          = "play-30"
+  val bootstrapVersion: String     = "8.5.0"
+  val hmrcMongoVersion: String     = "1.7.0"
 
-  val compile = Seq(
-    "uk.gov.hmrc"                  %% s"bootstrap-backend-$playVersion"    % bootstrapVersion,
-    "org.typelevel"                %% "cats-core"                  % "2.10.0",
-    "com.beachape"                 %% "enumeratum"                 % "1.7.3",
-    "ai.x"                         %% "play-json-extensions"       % "0.42.0",
-    "com.fasterxml.jackson.module" %% "jackson-module-scala"       % "2.15.3",
-    "com.github.andyglow"          %% "scala-jsonschema"           % "0.7.11",
-    "uk.gov.hmrc.mongo"            %% s"hmrc-mongo-$playVersion"   % hmrcMongoVersion
+  val compile: Seq[ModuleID] = Seq(
+    "uk.gov.hmrc"                  %% s"bootstrap-backend-$playVersion"   % bootstrapVersion,
+    "org.typelevel"                %% "cats-core"                         % "2.10.0",
+    "com.beachape"                 %% "enumeratum"                        % "1.7.3",
+    "ai.x"                         %% "play-json-extensions"              % "0.42.0",
```

Comment:
We have to get rid of this play-json-extensions.play-json-extensions
play-json-extensions is not compatible. Too old and no recent release.

Reply:
If I take it out I get a Json.format error if case class has 22 fields because in the model IndividualDetails there is a case class called Indicators that has 26 fields, I can changed it to     "com.github.arturopala"        %% "play-json-extensions"              % "1.0.0" but that's still play-json-extensions. Do you know another library I could use for this?

Reply:
Removed - manually constructed read and writes for the large Indicator object